Gaijinizer patch/Skills.txt

```diff
@@ -8261,20 +8333,21 @@ Beast Claw (Serious)#afrg
 > BEGIN STRING
 風属性の超特大ダメージを与える。
 これを覚えるのには強大な魔力を操れる必要がある。
-> CONTEXT: Skills/749/description < UNTRANSLATED
-
+> CONTEXT: Skills/749/description
+Deals massive wind damage. Requires a significant
+amount of magic power in order to learn.#Ron
 > END STRING
 
 > BEGIN STRING
 サイガ・ガルダ
-> CONTEXT: Skills/749/name < UNTRANSLATED
-
+> CONTEXT: Skills/749/name
+Psiga Garda#CheckLater
 > END STRING
```

Comment:
I think "Garuda" would be the proper spelling here, after the mythological being.